website/docs/r/downtime.html.markdown

```diff
@@ -47,7 +47,7 @@ resource "datadog_downtime" "foo" {
 
 The following arguments are supported:
 
-* `scope` - (Required) A list of items to apply the downtime to, e.g. host:X
+* `scope` - (Required)  The scope(s) to which the downtime applies, e.g. host:app2. Provide multiple scopes as a comma-separated list, e.g. env:dev,env:prod. The resulting downtime applies to sources that matches ALL provided scopes (i.e. env:dev AND env:prod), NOT any of them.
```

Comment:
As a user of the terraform `datadog_downtime` resource, `scope` is not a comma-separated list, but [really a list of strings](https://github.com/DataDog/terraform-provider-datadog/blob/v2.13.0/datadog/resource_datadog_downtime.go#L133-L138).

How that list is converted by the terraform provider to be sent to and read from the DataDog API is an implementation detail that should not be exposed, and this is confusing.